.editorconfig

```diff
@@ -94,7 +94,8 @@ dotnet_style_object_initializer = true:suggestion
 dotnet_style_collection_initializer = true:suggestion
 dotnet_style_explicit_tuple_names = true:suggestion
 dotnet_style_coalesce_expression = true:suggestion
-dotnet_style_null_propagation = true:suggestion
+dotnet_style_null_propagation = false:suggestion   # Turning off given new warnings that came with the new analyzers and we don't want to take the risk of changing those now
+dotnet_style_prefer_compound_assignment = false:suggestion
```

Comment:
I think I have all of these fixed in the main branch, so I don't think we need to disable the rules here.

Reply:
We've generally also been controlling such analyzers via https://github.com/dotnet/runtime/blob/main/eng/CodeAnalysis.src.globalconfig.